src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiActiveModulesUI.java

```diff
@@ -32,15 +32,16 @@ public void renderComponent(me.zeroeightsix.kami.gui.kami.component.ActiveModule
         GL11.glEnable(GL11.GL_BLEND);
         GL11.glEnable(GL11.GL_TEXTURE_2D);
 
+        activeMods = MODULE_MANAGER.getModuleT(ActiveModules.class);
+        
         FontRenderer renderer = Wrapper.getFontRenderer();
-        List<Module> mods = MODULE_MANAGER.getModules().stream()
-                .filter(Module::isEnabled)
-                .filter(Module::isOnArray)
-                .sorted(Comparator.comparing(module -> renderer.getStringWidth(module.getName() + (module.getHudInfo() == null ? "" : module.getHudInfo() + " ")) * (component.sort_up ? -1 : 1)))
-                .collect(Collectors.toList());
+	    List<Module> mods = MODULE_MANAGER.getModules().stream()
+		    	.filter(Module::isEnabled)
+	    		.filter(Module -> (activeMods.hidden.getValue() ? 1 == 1 : Module.isOnArray()))
```

Comment:
Fam why is there a 1==1 lmao

Reply:
its a place holder I couldn't think of what else to put there

Reply:
`true`? or just don't put anything? I'll take a look at this on a computer later

Reply:
I can't believe I didn't think of `true` lol